catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/NitfHeaderAttribute.java

```diff
@@ -108,220 +173,195 @@
+    public static final NitfHeaderAttribute STANDARD_TYPE_ATTRIBUTE  = new NitfHeaderAttribute(
+            STANDARD_TYPE,
             "STYPE",
             NitfHeader::getStandardType,
-            BasicTypes.STRING_TYPE,
-            "");
+            BasicTypes.STRING_TYPE);
 
-    public static final NitfHeaderAttribute FILE_DECLASSIFICATION_EXEMPTION =
-            new NitfHeaderAttribute("fileDeclassificationExemption",
+    public static final NitfHeaderAttribute FILE_DECLASSIFICATION_EXEMPTION_ATTRIBUTE =
+            new NitfHeaderAttribute(FILE_DECLASSIFICATION_EXEMPTION,
                     "FSDCXM",
                     header -> header.getFileSecurityMetadata()
                             .getDeclassificationExemption(),
-                    BasicTypes.STRING_TYPE,
-                    "");
+                    BasicTypes.STRING_TYPE);
 
-    public static final NitfHeaderAttribute FILE_DECLASSIFICATION_TYPE = new NitfHeaderAttribute(
-            "fileDeclassificationType",
+    public static final NitfHeaderAttribute FILE_DECLASSIFICATION_TYPE_ATTRIBUTE  = new NitfHeaderAttribute(
+            FILE_DECLASSIFICATION_TYPE,
             "FSDCTP",
             header -> header.getFileSecurityMetadata()
                     .getDeclassificationType(),
-            BasicTypes.STRING_TYPE,
-            "");
+            BasicTypes.STRING_TYPE);
 
-    public static final NitfHeaderAttribute FILE_DECLASSIFICATION_DATE = new NitfHeaderAttribute(
-            "fileDeclassificationDate",
+    public static final NitfHeaderAttribute FILE_DECLASSIFICATION_DATE_ATTRIBUTE  = new NitfHeaderAttribute(
+            FILE_DECLASSIFICATION_DATE,
             "FSDCDT",
             header -> header.getFileSecurityMetadata()
                     .getDeclassificationDate(),
-            BasicTypes.STRING_TYPE,
-            "");
+            BasicTypes.STRING_TYPE);
 
-    public static final NitfHeaderAttribute FILE_DOWNGRADE =
-            new NitfHeaderAttribute("fileDowngrade",
-                    "FSDG",
-                    header -> header.getFileSecurityMetadata()
-                            .getDowngrade(),
-                    BasicTypes.STRING_TYPE,
-                    "");
+    public static final NitfHeaderAttribute FILE_DOWNGRADE_ATTRIBUTE  = new NitfHeaderAttribute(
+            FILE_DOWNGRADE,
+            "FSDG",
+            header -> header.getFileSecurityMetadata()
+                    .getDowngrade(),
+            BasicTypes.STRING_TYPE);
 
-    public static final NitfHeaderAttribute FILE_DOWNGRADE_DATE = new NitfHeaderAttribute(
-            "fileDowngradeDate",
+    public static final NitfHeaderAttribute FILE_DOWNGRADE_DATE_ATTRIBUTE  = new NitfHeaderAttribute(
+            FILE_DOWNGRADE_DATE,
             "FSDGDT",
             header -> header.getFileSecurityMetadata()
                     .getDowngradeDate(),
-            BasicTypes.STRING_TYPE,
-            "");
+            BasicTypes.STRING_TYPE);
```

Comment:
For these date ones, I am hesitant to change them to BasicTypes.DATE_TYPE. These getter methods are returning strings. We have a helper method to convert CSDIDA date formats to a Date object, but I do not think, and do not know, that all formats on all TREs and NITF Header dates are the same format. I think we should leave this as is for now, and that this would potentially be worth another ticket.